bin/i18n/sync-out.rb

```diff
@@ -225,8 +225,12 @@ def serialize_i18n_strings(level, strings)
 
 def distribute_course_content(locale)
   locale_strings = {}
+  locale_dir = File.join("i18n/locales", locale)
+
+  Dir.glob(File.join(locale_dir, "course_content/**/*.json")) do |course_strings_file|
+    relative_path = course_strings_file.delete_prefix(locale_dir)
```

Comment:
Does this end up deleting strings that aren't changed?

For example, if coursea-2017 has been translated already but we get translations for coursea-2020 won't we skip adding the coursea-2017 strings to `locale_strings`?

Reply:
oh, good catch!

Reply:
Because this work was timeboxed, I'm just going to skip this optimization in this PR, and explore implementing it better as a follow-up